app/models/like.rb

```diff
@@ -1,4 +1,14 @@
 class Like < ApplicationRecord
-  belongs_to :author
-  belongs_to :post
+  belongs_to :user, foreign_key: :author_id
+  belongs_to :post, foreign_key: :post_id
+
+  def add_count
+    post.increment!(:likes_counter)
+    puts "Likes count: #{user.likes_counter}"
+  end
+
+  def remove_count
+    post.decrement!(:likes_counter)
+    puts "Likes count: #{user.likes_counter}"
+  end
```

Comment:
[OPTIONAL]
- You can consider the optional suggestion I made on `post.rb` about using [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html)